projects/cloud_feedbacks/highlights.md

```diff
@@ -4,6 +4,9 @@ title: Cloud Feedbacks
 ---
 # Identifying Robust Cloud Feedbacks in Observations and Models 
 
+![highlight]({{site.baseurl}}/projects/cloud_feedbacks/highlight.svg){:height="200px" width="720px"}
```

Comment:
```suggestion
![highlight]({{site.baseurl}}/projects/cloud_feedbacks/highlight.svg){:height="200px" width="200px" style="display: block; margin-left: auto; margin-right: auto;"}
```

Reply:
This should resize and align highlight.svg similar to binary.svg.

Reply:
@mzelinka You can apply the same change to the binary image in data.md so you don't need to directly edit the SVG file.